src/app/shared/components/number-input/number-input.component.ts

```diff
@@ -22,6 +22,9 @@ export class NumberInputComponent implements ControlValueAccessor {
   @Input() public placeholder?: string
 
   @Input() public helpText: string
+  @Input() public isRequired?: boolean
+  @Input() public hasError?: boolean
```

Comment:
This attribute should be deleted. Refer to my comment on `TextInputComponent`.

Reply:
Fixed